pandas/io/common.py

```diff
@@ -317,7 +317,8 @@ def _get_handle(path_or_buf, mode, encoding=None, compression=None,
         Supported compression protocols are gzip, bz2, zip, and xz
     memory_map : boolean, default False
         See parsers._parser_params for more information.
-
+    is_text : boolean, default True
+        whether file/buffer is in text format (csv, json, etc.), or in binary mode (pickle, etc.)
```

Comment:
@goldenbull The preferred coding style is not to use `\`, but use paratheses to have line continuation